models/src/main/scala/coop/rchain/models/rholang/RhoType.scala

```diff
@@ -165,14 +165,19 @@ object RhoType {
     def apply(gprivateBytes: ByteString): Par  = apply(GPrivate(gprivateBytes))
   }
 
-  type RhoUnforgeable = Unforgeable.type
-  object Unforgeable {
+  type RhoUnforgeable = RhoUnforgeable.type
+  object RhoUnforgeable {
     def unapply(p: Par): Option[GUnforgeable] = p.singleUnforgeable()
 
     def apply(unforgeable: GUnforgeable): Par = unforgeable
   }
 
-  type RhoExpression = Expression.type
+  type RhoExpression = RhoExpression.type
+  object RhoExpression {
+    def unapply(p: Par): Option[Expr] = p.singleExpr()
+
+    def apply(expr: Expr): Par = expr
+  }
 
   object SysAuthToken {
```

Comment:
Should `SysAuthToken` type also be prefixed with `Rho`?

Reply:
I thunk yes